include/libpmemfile-posix.h

```diff
@@ -204,6 +204,9 @@ typedef struct flock pmemfile_flock_t;
 
 #define PMEMFILE_AT_CWD ((PMEMfile *)(((unsigned char *)0) - 1))
 
+/* guaranteed to return at least 1 for any opened pool */
+unsigned pmemfile_pool_root_count(const PMEMfilepool *);
```

Comment:
const for pool pointer is IMO an overkill.

Reply:
Left it there by accident.